build/build.go

```diff
@@ -87,7 +101,28 @@ func Import(path string, mode build.ImportMode, installSuffix string, buildTags
 		}
 	}
 
-	return pkg, nil
+	jsFiles, err := jsFilesFromDir(pkg.Dir)
+	if err != nil {
+		return nil,err
+	}
+
+	return &PackageData{Package: pkg, JSFiles: jsFiles}, nil
+}
+
+// ImportDir is like Import but processes the Go package found in the named
+// directory.
+func ImportDir(dir string, mode build.ImportMode) (*PackageData, error) {
+	pkg,err := build.ImportDir(dir, mode)
+	if err != nil {
+		return nil,err
```

Comment:
> You have a sharp eye.

Thanks. :)

> Am I imagining things, or didn't this cause CI failures at some point in the past?

I am guessing it might've done that, but it might've been lost in the transition from Travis CI to circle CI.

> Regardless, should circle.yml be configured to fail for gofmt issues?

Agreed, opened #339.